buildscripts/observability-test/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+cd "$(dirname "$0")"/../..
+
+# TODO(stanleycheung): replace positional parameters with explicit parameters
```

Comment:
Talked offline, and I do really want this. The test framework would pass in `--server_host=blah` and such as arguments, and this script would handle any number of arguments (beyond the client/server first one). But that change will happen after this is initially merged.